ext/pgsql/pgsql_arginfo.h

```diff
@@ -652,3 +652,4 @@ static const zend_function_entry ext_functions[] = {
 	ZEND_FE(pg_select, arginfo_pg_select)
 	ZEND_FE_END
 };
+
```

Comment:
Side note: We should probably get rid of this trailing newline if no classes are defined...

Reply:
Yeah, I'll try to do something about it in the followup I promised.